packages/plugin-ext/src/hosted/node/hosted-plugin-manager.ts

```diff
@@ -209,6 +214,19 @@ export abstract class AbstractHostedPluginManager implements HostedPluginManager
         });
     }
 
+    protected logStdOut(data: string | Buffer) {
+        this.hostedPluginSupport.sendLog({
+            data: data.toString(),
+            type: LogType.Info
+        });
+    }
+
+    protected logStdErr(data: string | Buffer) {
```

Comment:
You can use one method to send logs(to prevent code duplication) with second parameter `LogType`. But up to you.

Reply:
I did it for shorted use in the event listeners but ok, agree. Will remove them